### DISCUSSION TITLE
请问lskyproV2的token怎么获取，显示405错误

### DISCUSSION THREAD

**Discussion by xxdccLove** [lsky-org/lsky-pro#376] [灌水]
请问lskyproV2的token怎么获取，显示405错误

**Comment by xxdccLove** [comment]
<html><body>
<li>登录到一个 V2 版本的兰空图床，比如 <a href="https://dogimg.com" target="_blank">https://dogimg.com</a> 进入 API 接口页面 <a href="https://dogimg.com/api" target="_blank">https://dogimg.com/api</a> 查看获取一个 token 的方式</li>
<li>使用 postman 之类的 api 调试工具发起一个 http post 请求即可生成一个 token，请求时的细节如下
<ol>
<li>请求 url: <code>https://dogimg.com/api/v1/tokens</code></li>
<li>请求方法: POST</li>
<li>设置请求头 <code>Accept</code> 的值为 <code>application/json</code></li>
<li>请求体中使用 json 语法填入邮箱和密码
<pre><code class="hljs json">





</code></pre></li>
<li>请求成功后得到返回信息中的 <code>token</code>，使用 <code>Bearer</code> 拼接拿到的 token 作为 <code>Auth token</code> 填入 PicGo 设置中。注意 Bearer 和 返回的 token 之间有个空格，请严格按照格式填写，程序不会校验这个细节</li>
</body>
</html>登录到一个 V2 版本的兰空图床，比如 https://dogimg.com/ 进入 API 接口页面 https://dogimg.com/api 查看获取一个 token 的方式
使用 postman 之类的 api 调试工具发起一个 http post 请求即可生成一个 token，请求时的细节如下

    请求 url: https://dogimg.com/api/v1/tokens
    请求方法: POST
    设置请求头 Accept 的值为 application/json
    请求体中使用 json 语法填入邮箱和密码

{

"email": "your_username@example.com",

"password":"your_password"

}

请求成功后得到返回信息中的 token，使用 Bearer 拼接拿到的 token 作为 Auth token 填入 PicGo 设置中。注意 Bearer 和 返回的 token 之间有个空格，请严格按照格式填写，程序不会校验这个细节

**Reply by DAMYSDS** [reply]
@dongguajun-none请问使用 Bearer 拼接拿到的 token 怎么操作

**Reply by xxdccLove** [reply]
直接Bearer token就可以，中间有个空格，可以参考我的博客
https://xxdcc.life/index.php/archives/6/

**Comment by cold8888** [comment]
可以直接使用我博客的这种方法获取token，输入账号密码直接获取https://www.coldyun.cn/archives/140.html
